### ISSUE TITLE
WCAG2ICT Key Term to modify: "Document"

### ISSUE THREAD

**Issue by jamieherrera** [w3c/matf#87]
Should we also include "Document" for WCAG2Mobile?
https://w3c.github.io/wcag2ict/#document

Key Terms Intro:

> Terms defined and used in WCAG2ICT are applicable only to the interpretation of the guidance in this document. The particular definitions should not be interpreted as having applicability to situations beyond the scope of WCAG2ICT. Further information on usage of these terms follows.

> The term document, as used in WCAG2ICT, has the meaning below:
> 
> document (as used in WCAG2ICT)
> assembly of [content](https://w3c.github.io/wcag2ict/#content-on-and-off-the-web), such as a file, set of files, or streamed media that functions as a single item rather than a collection, that is not part of software and that does not include its own user agent
> 
> Note 1
> A document always depends upon a user agent to present its content to the user.
> 
> Note 2
> Letters, spreadsheets, emails, books, pictures, presentations, and movies are examples of documents.
> 
> Note 3
> Software configuration and storage files such as databases and virus definitions, as well as computer instruction files such as source code, batch/script files, and firmware, are examples of files that function as part of [software](https://w3c.github.io/wcag2ict/#software) and thus are not examples of documents. If and where software retrieves “information and sensory experience to be communicated to the user” from such files, it is just another part of the content that occurs in software and is covered by WCAG2ICT like any other parts of the software. Where such files contain one or more embedded documents, the embedded documents remain documents under this definition.
> 
> Note 4
> A collection of files zipped together into an archive, stored within a single virtual hard drive file, or stored in a single "encrypted file system" file, do not constitute a single document.
> 
> Note 5
> Anything that can present its own content without involving a user agent, such as a self-playing book, is not a document but is software.
> 
> Note 6
> A single document may be composed of multiple files such as the video content, closed caption text, etc. This fact is not usually apparent to the end-user consuming the document / content. This is similar to how a single web page can be composed of content from multiple URIs (e.g. the page text, images, the JavaScript, a CSS file etc.).
> 
> Example: An assembly of files that represented the video, audio, captions, and timing files for a movie would be a document.
> 
> Counterexample: A binder file used to bind together the various exhibits for a legal case would not be a document.

**Comment by jamieherrera** [comment]
I think we could probably drop this one.

**Comment by detlevhfischer** [comment]
@jamieherrera There is a summary of the views subgroup now to be presented soon:https://docs.google.com/presentation/d/1cBLfSLZLq_mjICN9IKWD4-4PIOxQQzwvdRBWzFBnsyA/edit#slide=id.p
(you were asking somewhere, cannot find where...

**Comment by JJdeGroot** [comment]
Discussed in MATF meeting on August 27.

<details>
<summary>27 August 2025</summary>

[Source](https://www.w3.org/2025/08/27-matf-minutes.htmlfa6c)

~~~text
JJ: last item of today

JJ: document definition

JJ: do a poll on

JJ: drop the term document

JJ: in WCAG2ICT they differentiate between document and software

JJ: we wont need the document definition

JJ: non web document defintion

JJ: dont really need document definition in our guidance

<JJ> Poll: drop document definition in WCAG2Mobile?

<Jamie> +1

JJ: put plus one if you agree to drop, or put -1 or 0 if neutral, if don't agree then please explain, etc.

<Joe_Humbert> +1

<pauljadam> +1

<Tanya> +1

<Tim> +1

<shoobe01> 0

<Carol> +1

<GleidsonRamos> +1

JJ: 5 votes in so far

JJ: 9 participants minus JJ as chair

JJ: 8 votes

ACTION: Group is in favor of dropping the document term (7 times +1 and 1 time 0)
~~~
</details>

Group is in favor of dropping the document term (7 times +1 and 1 time 0)
